scripts/test-unix.sh

```diff
@@ -14,7 +14,7 @@ function cleanup {
         echo "Killing client with pid ${testPid}"
         kill -9 ${testPid}
         echo "Killing start-rc.sh with pid ${rcPid}"
-        kill ${rcPid}
+        kill -9 ${rcPid}
```

Comment:
If the process with PID `rcPid` doesn't exist the script exits with error. We can append something like `|| true` to prevent that.